Gopkg.toml

```diff
@@ -0,0 +1,44 @@
+[[constraint]]
+  name = "cloud.google.com/go"
+  version = "0.21.0"
+
+[[constraint]]
+  branch = "master"
+  name = "git.apache.org/thrift.git"
+  source = "github.com/apache/thrift"
+
+[[constraint]]
+  name = "github.com/golang/protobuf"
+  version = "1.0.0"
+
+[[constraint]]
+  name = "github.com/openzipkin/zipkin-go"
+  version = "0.1.0"
+
+[[constraint]]
+  name = "github.com/prometheus/client_golang"
+  version = "0.8.0"
+
+[[constraint]]
+  branch = "master"
+  name = "golang.org/x/net"
+
+[[constraint]]
+  branch = "master"
+  name = "golang.org/x/oauth2"
+
+[[constraint]]
+  branch = "master"
+  name = "google.golang.org/api"
+
+[[constraint]]
+  branch = "master"
+  name = "google.golang.org/genproto"
+
+[[constraint]]
+  name = "google.golang.org/grpc"
+  version = "1.11.3"
```

Comment:
I think we should default to specifying major version ranges for everything, so: version = "^1.11.3" here

Reply:
This is equivalent to `^1.11.3` https://golang.github.io/dep/docs/Gopkg.toml.html#version.